articles/firewall-manager/migrate-to-policy.md

```diff
@@ -25,11 +25,11 @@ Modify the following script to migrate your firewall configuration.
 
 ```azurepowershell
 #Input params to be modified as needed
-$FirewallName = "AZFW"
-$ResourceGroupName = "AzFWMigrateRG"
-$PolicyName = "fwp9"
-$Location = "WestUS"
-
+$FirewallResourceGroup = "fw-rg"
+$FirewallName = "azfw"
+$FirewallPolicyResourceGroup = "fwpolicy-rg"
```

Comment:
Let's have a single resource group for both the firewall and firewall policy. Having 2 RGs one for firewall and one for firewall policy is a bit of unnecessary overhead. I would recommend using the existing rg name "AzFWMigrateRG"

Reply:
Fixed